Specs/TestSpec/generated/Swift/Sources/Models/Zoo.swift

```diff
@@ -7,31 +7,17 @@ import Foundation
 
 public class Zoo: APIModel {
 
-    public var inlineAnimal: InlineAnimal?
+    public var allOfDog: Dog?
 
-    public var inlineAnimals: [InlineAnimals]?
-
-    public var schemaAnimals: [SingleAnimal]?
+    public var anyOfDog: Dog?
 
-    public class InlineAnimal: Animal {
```

Comment:
@yonaskolb  - This type is no longer generated but I think that is expected under this change.